src/05-derivatives/02-common-data-types.md

```diff
@@ -8,7 +8,7 @@ Template:
 <pipeline_name>/
     sub-<participant_label>/
         <datatype>/
-            <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+            <source_keywords>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<ext>
```

Comment:
From previous discussion, it looks like perhaps this ought to be moved inside 03-imaging...